specs/035-ci-add-sdd/enforcement.md

```
@@ -0,0 +1,227 @@
+```bash
+# Update from main
+git checkout main
+git pull origin main
+
+# Create enforcement branch
+git checkout -b feature/enable-ast-grep-enforcement
+```
+
+### 2. Modify CI Workflow
+
+Edit `.github/workflows/ci.yml`:
```

markdownlint-cli2 (0.18.1)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)